CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 * Ref: Make `WidgetsFlutterBinding.ensureInitialized();` the first thing the Sentry SDK calls.
 * Bump: Sentry-cocoa to 6.0.12
 * Feat: Respect FlutterError silent flag #248
+* Documentation: Added FlutterError.onError limitations
```

Comment:
```suggestion
```

Reply:
removed the changelog item cus its meant only for changes in the source code.

Reply:
Okay, I thought @bruno-garcia comment meant, that I should add a changelog.